dashboard/src/components/resource-list/BaseFilterPanel.js

```
@@ -7,11 +7,13 @@
 
 import {
   Accordion,
+  Box,
   Divider,
   FormControlLabel,
   TextField,
   Typography,
 } from '@mui/material';
+import { styled } from '@mui/system';
 import { Box, styled } from '@mui/system';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Box'. Consider to delete it or rename it.

'Box' is defined here:


(lint/suspicious/noRedeclare)